nemo_text_processing/inverse_text_normalization/es/verbalizers/time.py

```diff
@@ -36,11 +36,13 @@ def __init__(self):
         add_leading_zero_to_double_digit = (NEMO_DIGIT + NEMO_DIGIT) | (pynutil.insert("0") + NEMO_DIGIT)
 
         # hour may or may not include preposition ("la" or "las")
+        preposition = pynini.union("la ", "las ", "La ", "Las ")
```

Comment:
`article` not `preposition` (latter more confusing)

Reply:
where was my brain with this one -- changed